src/js/components/recipient/RecipientOverview.jsx

```diff
@@ -43,11 +43,10 @@ export default class RecipientOverview extends React.Component {
         else if (recipient.level === 'P') {
             // This is a parent recipient
             label = (
-                <a
-                    href={`#/recipient/${this.props.recipient.id}/?glossary=parent-duns`}
+                <span
                     className="recipient-overview__label recipient-overview__label_parent">
                     Parent Recipient <Glossary />
```

Comment:
Could you remove the Glossary icon as well (since it is used to indicate a link to the glossary elsewhere on the site)?